### PR TITLE
Convert ci-kubernetes-e2e-gce-1.3-1.4-upgrade jobs

### PR DESCRIPTION
**Pull request by xiangpengzhao**
#1230

This PR only includes a few jobs (ci-kubernetes-e2e-gce-1.3-1.4-upgrade-*).

cc @krzyzacy @fejta

### CODE REVIEW

**Comment by krzyzacy** [comment]
@k8s-bot ok to test

File: jobs/ci-kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new.env [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest-${version_new}
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new.env]: you want to resolve `${version_new}` here, same as below

Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new.env]: ah unit test did not catch this! I'll update the test :-)

Comment (xiangpengzhao) [inline jobs/ci-kubernetes-e2e-gce-1.3-1.4-upgrade-cluster-new.env]: Oh, I missed this point... :sweat_smile:  Will address. Thanks Sen!

**Comment by xiangpengzhao** [comment]
@krzyzacy inline comment addressed. PTAL :)

**Comment by krzyzacy** [comment]
@k8s-bot bazel test this

**Comment by krzyzacy** [comment]
/lgtm

you can watch the result on [testgrid](http://k8s-testgrid.appspot.com/google-1.3-1.4-upgrade-gke#gce-cvm-1.3-cvm-1.4-upgrade-master) and see how they work out. (it's a badly named tab...)

**Comment by xiangpengzhao** [comment]
Thanks Sen! I will learn the details and go on converting other jobs.

**Pull request merged**